samples/smoketest/pom.xml

```diff
@@ -0,0 +1,128 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
1. pom.xmls should inherit from our parent pom.
1. There should be no dependency version numbers.
1. The template for a pom is here: https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/template/azure-sdk-template/pom.xml

Here's an example:
https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/appconfiguration/azure-data-appconfiguration/pom.xml#L7-L12

Reply:
The Smoke Test is meant to behave like customer written code, and the customers are not going to use the parent pom.xml file.